ext/hal/openisa/vega_sdk_riscv/Kconfig

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2018 Foundries.io
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig HAS_VEGA_SDK_HAL
```

Comment:
"HAS_" options should not have any direct effect on the build when enabled.

It seems you have no need for a "HAS_" option, so you can simply rename this to "VEGA_SDK_HAL".